uxarray/_scrip.py

```diff
@@ -1,11 +1,121 @@
-from warnings import warn
+import xarray as xr
+import numpy as np
 
 
-def _read_scrip(filepath):
-    """This is just a function placeholder.
+def _to_ugrid(in_ds, outfile):
+    """If input dataset (ds) file is an unstructured SCRIP file, function will
+    reassign SCRIP variables to UGRID conventions in output file (outfile).
 
-    Not implemented.
+    Parameters
+    ----------
+    in_ds : :class:`xarray.Dataset`
+        Original scrip dataset of interest being used
+
+    outfile : :class:`xarray.Variable`
+        file to be returned by _populate_scrip_data, used as an empty placeholder file
+        to store reassigned SCRIP variables in UGRID conventions
+    """
+
+    if in_ds['grid_area'].all():
+
+        # Create Mesh2_node_x/y variables from grid_corner_lat/lon
+        # Turn latitude scrip array into 1D instead of 2D
+        corner_lat = in_ds['grid_corner_lat'].values
+        corner_lat = corner_lat.flatten()
+
+        # Return only the index of unique values to preserve order
+        lat_idx = np.unique(corner_lat, return_index=True)[1]
+        sort_lat = sorted(lat_idx)
+
+        # Create placeholder array for the sorted unique values
+        unique_lat = np.zeros(len(sort_lat))
+        for i in range(len(sort_lat) - 1):
+            unique_lat[i] = (corner_lat[sort_lat[i]])
```

Comment:
Is there a need for a for-loop, isn't this just doable with: 

`unique_lat  = corner_lat[sort_lat]`

Reply:
I tried this chunk without the for loop, and was getting a naming error, so have left it in it's current form. However, this comment did remind me that this chunk (and the lon version below) was implemented when I was looking into the writing function. On further consideration, I don't think there is a strong case to actually keep them overall. Originally, I wanted to preserve the ordering of the data points as I thought there was maybe a pattern I could use for the writing function, but that wasn't the case overall. That said, before removing this code completely, I wanted to double check that there wouldn't be other merit to preserving the data point index order instead of switching to the numpy default of re-ordering by data point value.